scripts/data_importer_service/data_import_service.py

```diff
@@ -63,9 +66,18 @@ def process_data_file(self, path, file_extension):
                 if isUnsupportedValue:
                     assessment[variable] = None
 
+        var_set = set(assessment_variables)
+        assessment_variables = list(var_set)
+        assessment_variables.sort()
```

Comment:
Nitpick (not blocking).  You could compress this into one step:

```suggestion
        var_set = set(assessment_variables)
        assessment_variables = sorted(var_set)
```

The built-in `sorted` function will accept any iterable and will return a list.

Reply:
https://github.com/AMP-SCZ/dpimport/pull/28/commits/f03fc80c1d6019fe2275c11f57b7d4a9c72d3f97